modules/xpfunmac.cmake

```diff
@@ -1032,8 +1032,10 @@ macro(xpSourceListAppend)
       list(SORT topdir) # sort list in-place alphabetically
       foreach(item ${topdir})
         xpGlobFiles(repoFiles ${item} *)
-        xpGlobFiles(fmtFiles ${item} *.c *.h *.cpp *.hpp *.cu *.cuh *.proto)
```

Comment:
Interestingly enough, this glob was taking around 20 seconds while the unrestrained glob on the line above was taking 2 seconds. (I measured this with about 60000 ignored files).  We are better off doing list operations as they were taking less than a second (I am not sure how to time it with more granularity.  I was using string(TIMESTAMP))